spec/models/like_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```

Comment:
- [ ] Custom methods in the models should have corresponding unit tests. Please test the `like_counter_update` method in your Like model